src/main/kotlin/com/hibob/academy/dao/PetDao.kt

```diff
@@ -1,17 +1,13 @@
+package com.hibob.academy.dao
 
-import com.hibob.academy.dao.*
 import jakarta.inject.Inject
 import org.jooq.DSLContext
 import org.jooq.Record
 import org.jooq.RecordMapper
 import org.jooq.impl.DSL
-import org.springframework.stereotype.Component
-import java.sql.Date
-import java.util.*
-import kotlin.random.Random
-
-@Component
+import org.springframework.stereotype.Repository
 
+@Repository
```

Comment:
Why not @AutoWired?

Reply:
I saw that usually @Repository is used in order to indicate that a class is a DAO